docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -250,3 +270,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
   "pollPeriod":600000
 }
 ```
+
+# Introspection
+
+Cached namespace lookups have introspection points at `/keys` and `/values` which return a complete set of the keys and values (respectively) in the lookup. Introspection to `/` returns the entire map. Introspection to `/version` returns the version indicator for the lookup, or a 404 on a race condition during a delete.
```

Comment:
why would delete be a race condition? If the lookup is deleted it seems normal to return 404